portal-api/src/v1/api.js

```diff
@@ -87,7 +87,7 @@ const deleteDeal = async (dealId) => {
       headers: headers.central,
     });
   } catch (error) {
-    return error;
```

Comment:
Please add console.error to all the return statements in following format:

`console.error('... %O', error)`;